x-pack/test/security_api_integration/tests/http_bearer/login.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { parse as parseCookie } from 'tough-cookie';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertestWithoutAuth');
+  const es = getService('es');
+
+  function extractSessionCookie(response: { headers: Record<string, string[]> }) {
+    const cookie = (response.headers['set-cookie'] || []).find((header) =>
+      header.startsWith('sid=')
+    );
+    return cookie ? parseCookie(cookie) : undefined;
+  }
+
+  async function createToken() {
+    const {
+      body: { access_token: accessToken },
+    } = await es.security.getToken({
+      body: {
+        grant_type: 'password',
+        username: 'elastic',
+        password: 'changeme',
+      },
+    });
+
+    return accessToken;
+  }
+
+  describe('login', () => {
+    it('does not accept bearer credentials as 401 status', async () => {
```

Comment:
Please let me know if this behavior seems correct. This is the behavior I intended, but I would like to know if there are any problems with not allowing logins for bearer tokens.